### ISSUE TITLE
Environment specs

### ISSUE THREAD

**Issue by wullish** [NVlabs/planercnn#15]
Hi @ztzhang , I didn't understand what you wrote here: https://github.com/NVlabs/planercnn/issues/3. You were using GCC version 6 but didn't worked because it does not work with version higher than 6 and then you switched to version 9 and worked? What?! Can you inform yours system specs, please?

I'm trying to use this pytorch repo with AWS Sagemaker p2.xlarge instance (rhel fedora 2018.3 with single GPU K80) which by default has CUDA 10.0 and GCC 4.8.5. So I've installed CUDA 9.0 and as GCC is version 4.8.5 by default (that's not higher than 6) but it didn't worked when trying to compile roialign.

The default configs (CUDA 10.0 and GCC 4.8.5) works with Mask R-CNN backbone using TensorFlow.

What I'm missing here @art-programmer ?

Thanks guys.

**Comment by art-programmer** [comment]
Sorry I have no experience with this particular setting. Could you please share the error messages? Also, maybe the original Mask R-CNN repo could give you some clues?
https://github.com/multimodallearning/pytorch-mask-rcnn

**Comment by wullish** [comment]
Everything is working now. Solved with CUDA 9.0 and GCC 5.3 as instructed. Apparently cudatoolkit=9.0 was not installed in my environment.
